offload/DeviceRTL/src/Reduction.cpp

```diff
@@ -196,8 +196,8 @@ int32_t __kmpc_nvptx_teams_reduce_nowait_v2(
   uint32_t NumThreads = omp_get_num_threads();
   uint32_t TeamId = omp_get_team_num();
   uint32_t NumTeams = omp_get_num_teams();
-  static unsigned SHARED(Bound);
-  static unsigned SHARED(ChunkTeamCount);
+  static unsigned __gpu_local Bound;
```

Comment:
which means these have lost their uninitialised, and now pretend to be zero initialised, which should be falling over if they aren't deadstripped